src/platform/packages/shared/kbn-esql-validation-autocomplete/BUILD.bazel

```diff
@@ -22,6 +22,7 @@ SRCS = glob(
 
 SHARED_DEPS = [
   "//src/platform/packages/shared/kbn-i18n",
+  "//src/platform/packages/shared/kbn-esql-types",
```

Comment:
Does it help to reduce the bundle size? Discover packages don't have `BUILD.bazel` files and I wonder if there is a benefit in adding them.